mscore/menus.cpp

```diff
@@ -1308,6 +1308,65 @@ void MuseScore::setBasicPalette()
       }
 
 //---------------------------------------------------------
+//   genCreateMenu
+//---------------------------------------------------------
+
+QMenu* MuseScore::genCreateMenu(QWidget* parent)
```

Comment:
That is correct. I moved the Add menu into mscore/musescore.cpp where the others were. So this whole thing should be removed.